var/spack/repos/builtin/packages/elsi/package.py

```diff
@@ -97,6 +110,12 @@ class Elsi(CMakePackage, CudaPackage):
     with when("+use_external_omm"):
         depends_on("omm")
         depends_on("matrix-switch")  # Direct dependency
+    with when("+dlaf"):
+        depends_on("dla-future-fortran")
+        conflicts("dla-future~cuda", when="+cuda")
+        conflicts("dla-future+cuda", when="~cuda")
+
+    conflicts("^pexsi@2:", when="@:2.11")
```

Comment:
is there any reason why you preferred to have this conflict not in related `when` scope? I was looking on how you dealt with `ntpoly` and I thought it looked the same use-case.

Reply:
Not really. Before, `ntpoly` didn't have a `when` context, so I put both conflicts together at the bottom. Later I moved the `ntpoly` one in the newly created context, but did not move the `pexsi` one.